lib/routes/bbc/index.ts

```diff
@@ -65,18 +65,26 @@ async function handler(ctx) {
 
                 const $ = load(response);
 
-                const description = new URL(item.link).pathname.startsWith('/news/av') ? item.content : utils.ProcessFeed($);
-
-                let section = 'sport';
-                const urlSplit = item.link.split('/');
-                const sectionSplit = urlSplit.at(-1).split('-');
-                if (sectionSplit.length > 1) {
-                    section = sectionSplit[0];
+                const path = new URL(item.link).pathname;
+
+                let description;
+
+                switch (true) {
+                    case path.startsWith('/sport'):
+                        description = item.content;
+                        break;
+                    case path.startsWith('/sounds/play'):
+                        description = item.content;
+                        break;
+                    case path.startsWith('/news/live'):
+                        description = item.content;
+                        break;
+                    default:
+                        description = utils.ProcessFeed($);
                 }
-                section = section[0].toUpperCase() + section.slice(1);
 
                 return {
-                    title: `[${section}] ${item.title}`,
+                    title: String(item.title),
```

Comment:
```suggestion
                    title: item.title,
```

Reply:
I have no idea where that came from...

Reply:
I guess you wrote 
```
title: `${item.title}`,
```
This can be fixed with [`no-useless-template-literals`](https://typescript-eslint.io/rules/no-useless-template-literals) but requires extending [`strict-type-checked`](https://typescript-eslint.io/users/configs/#strict-type-checked)